### PR TITLE
PLATUI-429: Capture page only when status is READY|PAGES_CAPTURED

### PR DESCRIPTION
**Pull request by ejayaraman**
- Pages will be captured only when the status is READY or PAGES_CAPTURED. 
- For all other status, a WARNING message is logged with the URL.

### CODE REVIEW

File: accessibility-assessment-service/app/routes/capturePage.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ router.post('/', (req, res, next) => {
   const pageDirectory = path.join(config.pagesDirectory, '' + body.timestamp)
   logData.pageHTML = logData.pageHTML.substr(0, 100) + '...'
   logData.files = Object.keys(logData.files)
+  const ALLOWED_STATUS = ["READY", "PAGES_CAPTURED"]
+
+  if(ALLOWED_STATUS.includes(global.status) === false) {
```
Comment (ejayaraman) [inline accessibility-assessment-service/app/routes/capturePage.js:22]: Is the preferred way?
` if(!ALLOWED_STATUS.includes(global.status))`

I thought explicitly checking for the false condition easy to read. Happy to change.

Comment (oscarduignan) [reply accessibility-assessment-service/app/routes/capturePage.js:22]: I'd normally go for ! but I'm happy either way

**Review by oscarduignan — APPROVED** [review]

File: accessibility-assessment-service/app/routes/capturePage.js (hunk 1 above)
Comment (kyle-bowden) [reply accessibility-assessment-service/app/routes/capturePage.js:22]: I think the way you have it is more readable.

Another way would be to make a function allowedStatusNotIncluded which is also very readable 

`const allowedStatusNotIncluded = (status) => {return !ALLOWED_STATUS.includes(status)}`

then 

`if(allowedStatusNotIncluded(global.status))`

Comment (ejayaraman) [reply accessibility-assessment-service/app/routes/capturePage.js]: Thanks, both. I have introduced a function as suggested above.

**Review by kyle-bowden — APPROVED** [review]

**Pull request merged**